lib/ansible/plugins/test/exists.yml

```
@@ -0,0 +1,22 @@
+DOCUMENTATION:
+  name: exists
+  author: Ansible Core
+  version_added: "2.5"
+  short_description: does the path exsit, follow symlinks
+  description:
+    - Check if the provided path maps to an existing filesystem object on the controller (localhost)
+    - Follows symlinks and checks the target of the symlink isteand of the link itself, use link or link_exists to check on the link.
```

```suggestion
    - Check if the provided path maps to an existing filesystem object on the controller (localhost).
    - Follows symlinks and checks the target of the symlink instead of the link itself, use the C(link) or C(link_exists) tests to check on the link.
```
(Would love to use `P(...)`, but we don't have semantic markup yet...)